src/lib/mollie/api/client.coffee

```diff
@@ -87,9 +87,21 @@ module.exports = class Client
 				body += data.toString();
 			);
 			response.on("end", ->
-				callback(JSON.parse(body))
+        if body.length
+          parsedBody = {}
+          try
+              parsedBody = JSON.parse(body)
+          catch e
+              console.log(e)
+
+          callback(parsedBody)
+        else
+          callback({})
```

Comment:
This may be true, but I believe the client is copying the models defaults (empty keys) on to the object that the API returns. So the resulting object would still be a valid model.

Check this line in the base class:
```callback(this.copy(body, new this.constructor.object));```